modules/core/resources/META-INF/smithy/presence.smithy

```diff
@@ -0,0 +1,9 @@
+$version: "2"
+
+namespace alloy
+
+/// Use this trait to mark some field as nullable. This is to make
+/// a distinction between an optional feel that's missing and one
+/// that's explicitly set to null.
+@trait()
```

Comment:
I think this should have a selector

Reply:
I was not sure what selector to use. I think applying that to trait shapes is irrelevant, so I used that.

Reply:
not sure what you mean - what I meant was that we should probably limit `@nullable` to just fields (or is there something else it can be used on?)

Reply:
At least current usages in `smithy-translate` sets `nullable` on top level shapes so I don't think limiting to `member` is right.

Reply:
uh oh. alright then